macOS/DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -238,9 +265,21 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
     }
 
     @objc func runUpdate() {
-        if let userDriver {
-            PixelKit.fire(DebugEvent(GeneralPixel.updaterDidRunUpdate))
+        guard let userDriver else { return }
+
+        PixelKit.fire(DebugEvent(GeneralPixel.updaterDidRunUpdate))
+
+        guard shouldForceUpdateCheck else {
             userDriver.resume()
+            return
+        }
+
+        userDriver.cancelAndDismissCurrentUpdate()
+        updater = nil
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
+            try? self?.configureUpdater(needsUpdateCheck: true)
+            self?.checkForUpdateRespectingRollout()
```

Comment:
Not sure if we should skip phased rollout group here. I feel like respecting rollout is a more logical choice, but upon restart the user could have their rollout group changed to be eligible for the actual most recent update and is again prompted for it...